website/docs/cloud-docs/workspaces/settings/drift-detection.mdx

```diff
@@ -0,0 +1,53 @@
+
+## Requirements
+
+Workspaces must must have the following settings to enable drift detection:
+- Terraform version 0.15.4+
+- [Remote execution mode](/cloud-docs/workspaces/settings#execution-mode) for Terraform runs
+
+## Enable Drift Detection
+
+All workspaces in an organization that meet the [requirements](#requirements) can use drift detection. To enable drift detection within a workspace:
+1. Go to the workspace and click **Settings > General**.
+1. Select **Enable** under **Drift Detection**.
+1. Click **Save settings**.
+
+Terraform Cloud starts drift detection immediately if there no active Terraform runs in the workspace. If you enable drift detection during a speculative plan, Terraform Cloud will start drift detection soon after completion. If you enable drift detection during other types of runs, Terraform Cloud will start the first automatic drift detection job in about 24 hours.
+
+Drift detection never interrupts or interferes with runs. Terraform Cloud begins drift detection assessments when at least 24 hours have passed since the last assessment and the workspace does not have a run in progress. You can also start new runs during a drift detection assessment. Starting a new run cancels the current assessment, and Terraform Cloud automatically starts the next one in 24 hours.
+
+Drift detection runs in the background and does not affect your concurrency limit. Terraform Cloud also monitors and controls drift detection concurrency, so it will not cause issues for large-scale deployments with thousands of workspaces. However, drift detection for organizations with a large number of workspaces may take longer to complete.
+
+## Workspace Drift Status
+
+After you enable drift detection, the workspace displays a drift status. A workspace can have one of the following drift statuses:
+
+- **Pending:** Terraform Cloud is performing a drift detection assessment for the workspace or there are no prior drift detection results. For example, a workspace will show this status when you enable drift detection during an active run.
+- **Drift:** Terraform Cloud has found that one or more real-world infrastructure resources do not match the workspace’s state file.
+-  **Drift Check Failed:** Terraform Cloud could not successfully complete the drift detection assessment for this workspace and will retry in 24 hours.
+- **Checked:** Terraform Cloud successfully completed the drift detection assessment for this workspace and found no drift.
+
+The workspace’s **Drift** tab contains details about the last drift detection assessment. If the workspace has drifted, Terraform Cloud displays a log file showing how the real-world infrastructure has changed from the latest version of the workspace’s state file.
```

Comment:
We don't show a log file, but a UI similar to the structured run output to display the changes.

I think we could get away with saying something like "Terraform Cloud displays how the real-world infrastructure has changed..."

Reply:
gotcha, I'll update this!